core/geometry/internaldocs/Spiral.md

```diff
@@ -0,0 +1,133 @@
+# Spirals
+
+## General Math
+
+A spiral is a curve that winds around a point (usually the origin), moving progressively closer or farther away from it.
+
+Mathematically, a spiral can be described in polar coordinates $(r,θ)$ where $r$ is the distance from the origin and $θ$ is the angle (in radians) from the x-axis. Therefore, a spiral can be defined by $r=f(θ)$ where $f(θ)$ describes how the radius grows as the angle increases.
+
+**Example:**
+
+*Archimedean Spiral:*
+$$r=a+bθ$$
+Parametric (Cartesian) form:
+$$x(θ)=(a+bθ)cosθ \\ y(θ)=(a+bθ)sinθ​$$
+
+*Logarithmic Spiral:*
+$$r=ae^{bθ}$$
+Parametric (Cartesian) form:
+$$x(θ)=ae^{bθ}cosθ \\ y(θ)=ae^{bθ}sinθ​$$
+
+**Note:** core-geometry does not currently support Archimedean or logarithmic spirals, and there are no plans to implement them. Their inclusion here is solely to illustrate the mathematical concepts behind spirals.
+-----------
+
+Another way to represent a spiral is via its curvature. In general, for a curve parameterized by curve length $\bar{s}$ we have:
+
+$$\frac{dθ}{d\bar{s}} = κ(\bar{s})$$
+
+where where $κ(\bar{s})$ is curvature. If $p$ is the position vector we have
+
+$$p(s) = (x'(\bar{s}),y'(\bar{s})) = (cosθ(\bar{s}), sinθ(\bar{s}))$$
+
+so integrating gives positions:
+
+$$x(\bar{s})= \int_{0}^{\bar{s}} cosθ(u)du \\ y(\bar{s})= \int_{0}^{\bar{s}} sinθ(u)du $$
```

Comment:
Consider putting the equations for x and y on different lines here and below.  They run together when rendered (at least in github's markup viewer).